### PR TITLE
various Python ports: remove EOL subports

### PR DESCRIPTION
**Pull request by reneeotten**
#### Description
This PR removes EOL Python subports for ports maintained by @stromnov ; none of them (should) have dependents left, so removing them will not break anything. 

Please not that this PR only removes subports and some occasional clean-up (i.e., removal of the default `platforms` field and `livecheck.type none` if there is only one `version` field in the Portfile). The CI will for sure timeout, but nothing will have changed regarding whether a port builds or not.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 10.15.7 19H1715 x86_64
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by reneeotten** [comment]
rebased again

**Comment by reneeotten** [comment]
thanks for approving the other PR @stromnov - how about this one?!

File: python/py-logbook/Portfile [hunk 1]
```diff
@@ -8,10 +8,9 @@ python.rootname     Logbook
 version             1.5.3
 revision            0
 categories-append   devel
-platforms           darwin
 license             BSD
 
-python.versions     27 35 36 37 38 39
+python.versions     27 37 38 39
```
Comment (stromnov) [inline python/py-logbook/Portfile:13]: Is there any reason to keep py27-logbook subport?

File: python/py-macfsevents/Portfile [hunk 2]
```diff
@@ -4,12 +4,12 @@ PortSystem          1.0
...
 
 name                py-macfsevents
+python.rootname     MacFSEvents
 version             0.8.1
 revision            0
-platforms           darwin
 license             BSD
 
-python.versions     27 35 36 37 38 39
+python.versions     27 37 38 39
```
Comment (stromnov) [inline python/py-macfsevents/Portfile:12]: Is there any reason to keep py27-macfsevents subport?

File: python/py-magic/Portfile [hunk 3]
```diff
@@ -8,11 +8,10 @@ python.rootname     python-magic
 version             0.4.24
 revision            0
 categories-append   devel
-platforms           darwin
 license             MIT
 supported_archs     noarch
 
-python.versions     27 35 36 37 38 39 310
+python.versions     27 37 38 39 310
```
Comment (stromnov) [inline python/py-magic/Portfile:14]: Is there any reason to keep py27-magic subport?

File: python/py-logbook/Portfile (hunk 1 above)
Comment (reneeotten) [reply python/py-logbook/Portfile:13]: yes, it still has a dependent, for which I didn't manage (yet) to update the Python version: 
```
% port echo depends:py27-logbook
pyfa
```

File: python/py-macfsevents/Portfile (hunk 2 above)
Comment (reneeotten) [reply python/py-macfsevents/Portfile:12]:
```
% port echo depends:py27-macfsevents
ninja-ide
```

File: python/py-magic/Portfile (hunk 3 above)
Comment (reneeotten) [reply python/py-magic/Portfile:14]:
```
% port echo depends:py27-magic
fdroidserver
alot
```

**Comment by reneeotten** [comment]
@stromnov hopefully I'll get to those at some point and will remove them as well if there are no dependents anymore. If you're okay with it, it's probably more efficient if I  just commit changes like this directly instead of opening a PR?

**Review by stromnov — APPROVED** [review]
Looks good to me.

**Pull request merged**